src/lib/email/create-ical-event.ts

```diff
@@ -8,27 +8,40 @@ type CreateIcalEventParams = {
   id?: string;
   summary: string;
   description?: string;
-  start: Date;
-  end: Date;
+  start: string | Date;
+  end: string | Date;
   location?: string;
 };
 
 export const createIcalEvent = (
   params: CreateIcalEventParams,
 ): ICalCalendar => {
   const calendar = ical({
-    name: "De Sering", // todo: don't hardcode our name anywhere
+    name: `${process.env.ORG_NAME}·Calendar` || "Event·Calendar",
```

Comment:
This does not work, have a look at `create-ical-event.spec.ts`. The output will be `undefined Calendar` if the env variable is missing.

```suggestion
    name: process.env.ORG_NAME ? `${process.env.ORG_NAME}·Calendar` : "Event·Calendar",
```